lib/ret/media_search.ex

```diff
@@ -62,38 +63,89 @@ defmodule Ret.MediaSearch do
     favorites_search(cursor, type, account_id, q)
   end
 
+  def search(%Ret.MediaSearchQuery{source: "sketchfab", cursor: cursor, filter: nil, collection: nil, q: q}) do
+    if q == nil || q == "" do
```

Comment:
this should be a separate match

Reply:
How can I match on nil and ""?

Reply:
you can use `when` https://hexdocs.pm/elixir/master/guards.html